Alcatraz/Controllers/ATZPluginWindowController.m

```diff
@@ -237,6 +251,11 @@ - (void)updatePredicate {
 
     [self.tableViewDelegate filterUsingPredicate:[NSCompoundPredicate andPredicateWithSubpredicates:predicates]];
     [self.tableView reloadData];
+
+	dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(0.1 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
+		// wait next run loop, otherwise there are no rows yet
+		[self loadImagesInVisibleRows];
+	});
```

Comment:
Why load the table and then fetch images instead of loading the images when the cell is loaded? There's a bit more delay before all the images have been cached.